.github/workflows/build.yaml

```diff
@@ -15,12 +15,31 @@ jobs:
       - name: Setup Linux Build Dependencies
         if: ${{ runner.os == 'Linux' }}
         run: sudo apt-get install ninja-build
+
+      - name: Run apt update
+        if: runner.os == 'Linux'
+        run: sudo apt update
```

Comment:
Nit: combine this with the package install into a single step. Less steps, less conditions.